.github/kubeval.sh

```
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-mkdir ./.bin
+mkdir -p ./.bin
```

why this change?

---

To suppress errors if the folder already exists.